wrappers/ortools.rb

```diff
@@ -774,8 +788,11 @@ def run_ortools(problem, vrp, services, points, matrix_indices, thread_proc = ni
         log 'Job killed', level: :fatal # Keep trace in worker
         raise OptimizerWrapper::JobKilledError
       else # Fatal Error
-        message = if @thread.value == 127
+        message = case @thread.value
+                  when 127
                     'Executable does not exist'
+                  when 137 # Segmentation Fault
+                    'Solver has met an unexpected error'
```

Comment:
I believe 137 is related to out-of-memory 

Exit Code 137: Indicates failure as container received SIGKILL (Manual intervention or ‘oom-killer’ [OUT-OF-MEMORY])
```suggestion
                  when 137 # Segmentation Fault
                    "Executable killed by 'oom-killer' [OUT-OF-MEMORY]"
```

Reply:
It could be a manual intervention OR the 'oom-killer'.

Nevertheless, I will correct it accordingly